draft-ietf-scitt-architecture.md

```diff
@@ -373,280 +369,155 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
-  ledger_alg: int ; type of verifiable data structure
-  service_uri: tstr; base URI of the transparency service, will be the issuer in the receipt CWT claim set
-  root_certificates: [ COSE_X509 ]
-  supported_apis: [ SCITT_Endpoint ]
-  registration_policies : Registration_Policy
-]
-~~~
+The Append-only Log is empty when the Transparency Service is initialized.
+The first entry that is added to the Append-only Log MUST be a Signed Statement including key material.
+The second set of entries are Signed Statements for additional domain-specific Registration Policy.
+The third set of entries are Signed Statements for Artifacts.
+From here on a Transparency Service is capable to check Signed Statements on registration via policy (that is at minimum key material and typically other Registration Policy) and is therefore in a reliable state to register Signed Statements about Artifacts or new Registration Policy.
 
 ### Registration Policies
 
-Authorization is needed prior to registration of Signed Statements to ensure completeness of an audit.
-A Transparency Service that registers valid Signed Statement offered by anonymous Issuers would provide limited to no value to Verifiers.
-More advanced use case will rely on the Transparency Service performing additional domain-specific checks before a Signed Statement is accepted.
-For example, some Transparency Services may validate the non-opaque content (payload) of Signed Statements.
+Registration Policies refer to the checks that are performed before a Signed Statement is registered to an Append-only Log, and a corresponding Receipt becomes available.
 
-Registration Policies refers to the checks that are performed before a Signed Statement is registered given a set of input values.
-This specification leaves the implementation of the Registration Policy to the provider of the Transparency Services and its users.
-
-A provider of a Transparency Service indicates what Registration Policy is used in a given deployment and inform its users about changes to the Registration Policy by issuing and registering configuration statements.
-
-As a minimum, a Transparency Service MUST authenticate the Issuer of the Signed Statement, which requires some form of trust anchor.
+As a minimum, a Transparency Service MUST authenticate the Issuer of Signed Statements, which requires a trust anchor in the form of an already registered Signed Statement including key material (see {{ts-initialization}}).
 As defined in {{RFC6024}}, "A trust anchor represents an authoritative entity via a public key and associated data.
 The public key is used to verify digital signatures, and the associated data is used to constrain the types of information for which the trust anchor is authoritative."
-The Trust Anchor may be a certificate, a raw public key or other structure, as appropriate.
-It can be a non-root certificate when it is a certificate.
-
-### Append-only Log Security Requirements
```

Comment:
That is a lot of deleted text here, which against weakens the definition of a TS significantly. If your expectation is that anything that supports Merkle Tree proof of inclusion will meet these requirements, I don't think that is the case as the COMETRE ID doesn't have finality or consistency requirements to create new algorithms.

Reply:
Or goes to the COMETRE I-D, of course. I forgot to mention that.